Source/ASDisplayNode.mm

```diff
@@ -58,6 +58,8 @@
   #define TIME_SCOPED(outVar)
 #endif
 
+#define ENABLE_NEW_EXIT_HIERARCHY_BEHAVIOR 0
```

Comment:
Would you be willing to add a comment describing this macro?

Reply:
Hi, you mean I added in code or here?
The new NEW_EXIT_HIERARCHY_BEHAVIOR is trying to merge non-rangeManaged with rangeManaged, so both range-managed and standalone nodes wait before firing their exit-visibility handlers, as UIViewController transitions now do rehosting at both start & end of animation. (that was to try to mitigate interface updating state when coalescing disabled)

Reply:
OK that makes sense. Could you add that on line 60 so that future developers will know the intention?

Reply:
Done